lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -1034,7 +1058,7 @@ def create_servers(action_handler, specs_and_options, parallelizer, &block)
       end
 
       # Create the servers in parallel
-      parallelizer.parallelize(by_bootstrap_options) do |bootstrap_options, machine_specs|
+      parallelizer.parallel_do(by_bootstrap_options) do |bootstrap_options, machine_specs|
```

Comment:
It's very strange that this change works at all.  `parallelize` returns a `ParallelEnumerable`, while `parallel_do` returns `nil`.